packages/web/src/components/sessions/SessionTerminal.tsx

```
@@ -45,9 +47,18 @@ const RESUMABLE_STATUSES = new Set(["done", "needs_input", "stuck", "errored", "
 const RECONNECT_BASE_DELAY_MS = 300;
 const RECONNECT_MAX_DELAY_MS = 1600;
 const RENDERER_RECOVERY_THROTTLE_MS = 120;
-const LIVE_TERMINAL_SCROLLBACK = 200000;
-const READ_ONLY_TERMINAL_SNAPSHOT_LINES = 200000;
+const LIVE_TERMINAL_SCROLLBACK = 50000;
+const LIVE_TERMINAL_SNAPSHOT_LINES = 1200;
+const READ_ONLY_TERMINAL_SNAPSHOT_LINES = 6000;
 const MANAGED_SCROLL_PRIVATE_MODES = new Set([1000, 1002, 1003, 1005, 1006, 1015, 1047, 1048, 1049]);
+const DEFAULT_REMOTE_POLL_INTERVAL_MS = 700;
+const BROWSER_TERMINAL_RESPONSE_PATTERNS = [
+  /\x1b\[(?:I|O)/g,
```

Biome (2.4.6)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)